openpype/settings/lib.py

```diff
@@ -1093,3 +1093,17 @@ def clear_metadata_from_settings(values):
     elif isinstance(values, list):
         for item in values:
             clear_metadata_from_settings(item)
+
+
+def get_versioning_start():
+    """Get anatomy versioning start"""
+    project_name = os.environ.get("AVALON_PROJECT")
+    version_start = 1
+    settings = get_anatomy_settings(project_name)
+    if "version_start_at_zero" not in settings["templates"]["defaults"]:
+        settings = get_default_anatomy_settings()
+
+    if settings["templates"]["defaults"]["version_start_at_zero"]:
+        version_start = 0
+
+    return version_start
```

Comment:
I feel like we should be redesigning this to allow taking in a `anatomy_settings` argument or whatever or perform some caching of sorts. The function is used in a quite a bit of places and we'd probably want to reduce how many times this queries the database? 


Or - easier to implement - we can maybe cache/store the value during runtime because we'd likely not want to change the version start value mid-session in a running maya session? :) In that scenario we only have to query database once per session.

Reply:
> Or - easier to implement - we can maybe cache/store the value during runtime because we'd likely not want to change the version start value mid-session in a running maya session? :) In that scenario we only have to query database once per session.

Yeah, I was looking into caching, but it would work quite different to other settings which are picked up while you have a Maya session open.

@iLLiCiTiT think its worth optimizing?